code/game/objects/structures/false_walls.dm

```diff
@@ -169,7 +169,7 @@
 					T.ChangeTurf(/turf/simulated/wall)
 				else
 					T.ChangeTurf(text2path("/turf/simulated/wall/mineral/[mineral]"))
-				if(mineral != "plasma")//Stupid shit keeps me from pushing the attackby() to plasma walls -Sieve
+				if(mineral != PLASMA)//Stupid shit keeps me from pushing the attackby() to plasma walls -Sieve

```

Comment:
inappropriate replacement (mineral IDs, those need to be refactored separately)
same for the rest of this file